test/integration/fleetscope/fleetscope_test.go

```diff
@@ -163,6 +163,25 @@ func TestFleetscope(t *testing.T) {
 					assert.Equal(gkeScopes, opGKEScopes.Get("name").String(), fmt.Sprintf("The GKE Namespace should be %s", gkeScopes))
 					assert.True(opGKEScopes.Exists(), "Namespace %s should exist", gkeScopes)
 				}
+				gkeMeshCommand := fmt.Sprintf("beta container fleet mesh describe --project %s", clusterProjectID)
+				pollMeshProvisioning := func(cmd string) func() (bool, error) {
+					return func() (bool, error) {
+						result := gcloud.Runf(t, cmd).Array()
+						if len(result) < 1 {
+							return true, nil
+						}
+						dataPlaneManagement := result[0].Get("membershipStates.servicemesh.dataPlaneManagement.state").String()
+						controlPLaneManagement := result[0].Get("membershipStates.servicemesh.controlPlaneManagement.state").String()
+						if dataPlaneManagement == "FAILED_PRECONDITION" || controlPLaneManagement == "FAILED_PRECONDITION" {
+							return false, fmt.Errorf("Service Mesh provisioning failed.")
+						} else if dataPlaneManagement == "PROVISIONING" || controlPLaneManagement == "PROVISIONING" {
+							return true, nil
```

Comment:
```suggestion
							return false, nil
```

Reply:
actually we need to run into membershipStates. Testing locally the solution.